### ISSUE TITLE
Add a parameter in (the Utopia) json ?

### ISSUE THREAD

**Issue by leodium** [cwmonkey/nms-expeditions#9]
Another thing ... Do you think it's possible to add parameters to the json that aren't "planned" at the outset?

Add the parameter in the Utopia json to be able to transfer the items at the end of the expedition?
`"HasAccessToNexus" = True `

https://steamcommunity.com/app/275850/discussions/0/4286936086674242135/?ctp=2#c4362372646221855039

**Comment by cwmonkey** [comment]
@leodium I tried setting this before, after and during the expedition and it didn't unlock the Anomaly.

Only way to get the Anomaly is to delete the `SEASON_DATA_CACHE.JSON` file and relaunch the game, or close the game, go online, and launch the game. This will start you on the Anomaly so you can copy your multitool/ship. Unfortunately you do not get a chance to transfer your items.

**Comment by leodium** [comment]
Yes, I tried too. But I didn't know if I was putting the right form and/or in the right place. It was worth a try anyway. It's even weird that some Expeditions can go there and not this one... Or a Polo/Nada parameter

**Comment by cwmonkey** [comment]
One thing I noticed is there is some setup in the .pak files that set certain things. There are even quests/missions that fire before you can do anything in the expedition (this is how they give you certain tech items at the start). My guess is it's somewhere in there where they lock you out of the Anomaly.

I tried looking for a milestone to unlock the Anomaly so I could encrypt it and add it to the end but no luck yet,

**Comment by Dark0080** [comment]
Hello guys
im just a noob at this stuff but i managed to find a way to enable the Nexus summoning via the json!!!

Search under Expedition content, for "HasAccessToNexus" and just replace the False to true!!!

Thank you guys for doing what youve done!!! I hope this helps someone :)

**Comment by leodium** [comment]
😱🤞🤞🤞

Would it be that simple??? I'm not on my computer to test... 🥴 I'll watch
tonight... Would it be that simple

**Comment by leodium** [comment]
@Dark0080 

Ah... Damn ... it has already been tested ... https://github.com/cwmonkey/nms-expeditions/issues/9#issuecomment-2025758598

Did it work for you?

**Comment by Dark0080** [comment]
@leodium 

Yes it worked for me
I was able to successfully summon the nexus and transfer my items back to my save

it's not going to be in "difficulty"

rather hidden way deep in the "expedition save state"
hmm I was using the NMS Save Editor... idk if any other way to read the JSON file is in a different format.

like i said, it is way deep in that section

I searched Nexus on the Exp Save State
found and changed 2 things
1st looked like a mission tag
"EnableNexus"
and I changed the variable to -1 as that matches the regular Player Save State for the same mission
that alone didn't unlock it however
so I looked and found
"hasaccesstonexus" a bit farther down the JSON and changed it to true

after that, I was able to summon the anomaly.

**Comment by leodium** [comment]
Argh ... Oki.

I'm afk ... but i want to look !?! 😅🤞🤞

**Comment by cwmonkey** [comment]
Thanks @Dark0080 - I meant to add a note about this a while back when it was mentioned on the Steam page but I forgot. The notes have been added to https://cwmonkey.github.io/nms-expeditions/

I also need to get around to finishing my online save editor so it can just do the fix for you easily. I just have to take the time to get around a weird encoding issue with stored x tech.

**Comment by cwmonkey** [comment]
A while back I added a patch to change the last milestone's reward to one that unlocks the Anomaly.
